README.md

```diff
@@ -6,37 +6,34 @@
 <br>
 <p align="center">
   <a href="https://www.ruby-lang.org/en/">
-    <img src="https://img.shields.io/badge/Ruby-v2.7.1-green.svg" alt="ruby version">
+    <img src="https://img.shields.io/badge/Ruby-v2.7.2-green.svg" alt="ruby version">
   </a>
   <a href="http://rubyonrails.org/">
     <img src="https://img.shields.io/badge/Rails-v6.0.3-brightgreen.svg" alt="rails version">
   </a>
-  <a href="https://travis-ci.com/thepracticaldev/dev.to">
-    <img src="https://travis-ci.com/thepracticaldev/dev.to.svg?branch=master" alt="Travis Status for thepracticaldev/dev.to">
+  <a href="https://travis-ci.com/forem/forem">
+    <img src="https://travis-ci.com/forem/forem.svg?branch=master" alt="Travis Status for forem/forem">
   </a>
-  <a href="https://codeclimate.com/github/thepracticaldev/dev.to/maintainability">
+  <a href="https://codeclimate.com/github/forem/forem/maintainability">
     <img src="https://api.codeclimate.com/v1/badges/ce45bf63293073364bcb/maintainability" alt="Code Climate maintainability">
   </a>
-  <a href="https://codeclimate.com/github/thepracticaldev/dev.to/test_coverage">
-    <img src="https://api.codeclimate.com/v1/badges/ce45bf63293073364bcb/test_coverage" alt="Code Climate test coverage">
+  <a href="https://codeclimate.com/github/forem/forem/trends/technical_debt">
+    <img src="https://img.shields.io/codeclimate/tech-debt/forem/forem" alt="Code Climate technical debt">
   </a>
-  <a href="https://codeclimate.com/github/thepracticaldev/dev.to/trends/technical_debt">
-    <img src="https://img.shields.io/codeclimate/tech-debt/thepracticaldev/dev.to" alt="Code Climate technical debt">
-  </a>
-  <a href="https://www.codetriage.com/thepracticaldev/dev.to">
-    <img src="https://www.codetriage.com/thepracticaldev/dev.to/badges/users.svg" alt="CodeTriage badge">
+  <a href="https://www.codetriage.com/forem/forem">
+    <img src="https://www.codetriage.com/forem/forem/badges/users.svg" alt="CodeTriage badge">
   </a>
   <img src="https://badgen.net/dependabot/forem/forem?icon=dependabot" alt="Dependabot Badge">
   <a href="https://gitpod.io/from-referrer/">
     <img src="https://img.shields.io/badge/setup-automated-blue?logo=gitpod" alt="GitPod badge">
   </a>
-  <a href="https://app.netlify.com/sites/devto/deploys">
+  <a href="https://app.netlify.com/sites/docsdevto/deploys">
```

Comment:
I figured out the name from the PR check. Can someone verify the UUID on the next line, maybe @Zhao-Andy?